src/common/constants.rs

```diff
@@ -1,5 +1,5 @@
-pub const GRAPHQL_ENDPOINT: &str = "https://graphql.minaexplorer.com";
-pub const REST_ENDPOINT: &str = "https://api.minaexplorer.com";
+pub const GRAPHQL_ENDPOINT: &str = env!("GRAPHQL_URL");
+pub const REST_ENDPOINT: &str = env!("REST_URL");
```

Comment:
Should these have trailing slashes?

We should document these somewhere.

Reply:
Nope, it shouldn't. I suppose we could do more logic around it